backend/api/consumers.py

```diff
@@ -135,6 +157,12 @@ def get_printer(self):
         ).printer
 
     @newrelic.agent.background_task()
+    @close_on_error
+    def connect(self):
```

Comment:
Any reason why we redefine `connect` here?

Reply:
Nothing else than consistency.